backend/config/initializers/rails_admin.rb

```diff
@@ -0,0 +1,29 @@
+RailsAdmin.config do |config|
+
+  config.authorize_with do
+    authenticate_or_request_with_http_basic('Admin Authentication') do |username, password|
+      admin = Client.find_by_id(1)
+      username == admin.name.downcase && admin.authenticate(password)
```

Comment:
And please don't do this `admin = Client.find_by_id(1)`

In your local database, client with name "Foo" can have the id of `1`. But let's say in production, it could be "Baz" who owns the id of `1`. In short, remove static queries whenever possible.